### PR TITLE
Make @types/node a devDependency

### PR DESCRIPTION
**Pull request by GioAc96**
## Changes

- Make `@types/node` a devDependency instead of a normal dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +63,8 @@
     "split2": "^4.1.0",
     "swagger-parser": "^10.0.3",
     "tap": "^16.1.0",
-    "typescript": "^4.6.4"
+    "typescript": "^4.6.4",
+    "@types/node": "^16.7.1"
```
Comment (ilteoood) [inline package.json:67]: It should be on top, as the other dependencies that starts with @

Comment (GioAc96) [reply package.json]: Fixed

**Comment by davidebianchi** [comment]
Is this PR correct? The `@types/node` dependency is used in index.d.ts, which is exported. This change could break the export of this types?

**Comment by GioAc96** [comment]
@davidebianchi you're probably correct (see link below). Closing the PR:

https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies

**Pull request closed without merging**